mithril-aggregator/src/configuration.rs

```diff
@@ -298,32 +302,6 @@ impl Configuration {
 
         Ok(all_discriminants)
     }
-
-    /// Create the deduplicated list of allowed signed entity types.
-    ///
-    /// By default, the list contains the MithrilStakeDistribution and the CardanoImmutableFilesFull.
-    /// The list can be extended with the configuration parameter `signed_entity_types`.
-    /// The signed entity types are defined in the [SignedEntityTypeDiscriminants] enum.
-    /// The signed entity types are discarded if they are not declared in the [SignedEntityType] enum.
```

Comment:
This comment could be added to the `parse_signed_entity_types_discriminants` function IMO